Namco/position_comparison.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import os
```

Comment:
Ezeket ha jól látom nem használjuk, így nem kellenek. Vagy van valami cél velük, amit nem látok?

Reply:
Sztem sem használjuk. Ctrl Alt L ezt is megjavítja (pycharmban van ez a billentyű parancs)

Reply:
Nem, csak ott maradtak és nem figyeltem javitottam